### PR TITLE
Jb53194

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the last commit belongs to this PR

File: src/plugins/mer/merbuildengineoptionswidget.cpp [hunk 1]
```diff
@@ -643,6 +644,7 @@ void MerBuildEngineOptionsWidget::onSrcFolderApplyButtonClicked(const QString &n
     buildEngine->virtualMachine()->lockDown(false, this, IgnoreAsynchronousReturn<bool>);
 
     if (ok) {
+        Core::DocumentManager::setProjectsDirectory(buildEngine->sharedSrcPath());
```
Comment (vigejolla) [inline src/plugins/mer/merbuildengineoptionswidget.cpp:647]: Wouldn't it make sense to first check if current projectsDirectory already happens to be under workspace?

Comment (martyone) [reply src/plugins/mer/merbuildengineoptionswidget.cpp]: Yup, done.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**